### PR TITLE
ACNA-274, ACNA-272 - support for aio-cli-config, remove paging

### PR DESCRIPTION
**Pull request by goya**
## Description

* add support for aio-cli-config
* add integration to view an integration
* add selected-integration to get current selected integration
* eliminate paging #36 
* display selected integration in list

## How Has This Been Tested?
100% test coverage

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by shazron** [comment]
Please have the separate feature fix (paging removal) as a separate PR next time for clarity and tracking.

**Comment by shazron** [comment]
All builds in the build matrix failed the first time, I restarted the build just in case it was a server one-time glitch.

**Comment by shazron** [comment]
I had to cancel the build for Windows on Node 8, because it was running for 12 minutes when it shouldn't take longer than 4. The other 5 builds have failed.

These are genuine build errors it seems, please see the Travis logs.

**Comment by shazron** [comment]
All Windows builds are failing (out of memory error).

**Comment by goya** [comment]
yeah saw that, might okay around a bit tomorrow.

File: package.json [hunk 1]
```diff
@@ -62,30 +60,11 @@
   },
   "repository": "adobe/aio-cli-plugin-console",
   "scripts": {
-    "eslint": "eslint src test",
+    "eslint": "eslint .",
     "test": "npm run eslint && npm run unit-tests",
-    "unit-tests": "jest --ci -w=2",
+    "unit-tests": "jest",
```
Comment (shazron) [inline package.json]: This is the reason the Windows builds are failing. If you restrict the number of workers to 2 (like the original line), they pass. Revert this line to make the build pass on Windows.

Comment (shazron) [reply package.json]: See #40 where I validate this

**Pull request merged**